custom_components/ai_thermostat/models/utils.py

```diff
@@ -34,22 +32,6 @@ def default_calibration(self):
 	new_calibration = float((float(self._cur_temp) - float(state.get('local_temperature'))) + float(state.get('local_temperature_calibration')))
 	return convert_decimal(new_calibration)
 
-
-async def evaluate_dampening(self, calibration):
-	state = self.hass.states.get(self.heater_entity_id).attributes
-	if (datetime.now() > (self.last_dampening_timestamp + timedelta(minutes=15))) and state.get(
-			'system_mode') is not None and self._target_temp is not None and self._cur_temp is not None and not self.night_status:
-		check_dampening = (float(self._target_temp) - 0.5) < float(self._cur_temp)
-		if check_dampening:
-			self.ignore_states = True
-			_LOGGER.debug("Dampening started")
-			await set_trv_values(self,'temperature', float(5))
-			await asyncio.sleep(60)
-			await set_trv_values(self,'temperature', float(calibration))
-			self.last_dampening_timestamp = datetime.now()
-			self.ignore_states = False
-
-
```

Comment:
This is not part of the PR's intended changes. Please remove this from your changeset :)